app/models/user.rb

```diff
@@ -11,12 +11,28 @@ class User < RedisStore
   attribute :last_name
   attribute :zip
 
+  # vaafi attributes
+  attribute :edipi
+  attribute :issue_instant
+  attribute :participant_id
+  attribute :ssn
```

Comment:
Yeah, this was just a matter of consistency. Not sure what the pros/cons are of different ways of declaring attributes.